nxviz/plots.py

```diff
@@ -338,10 +340,18 @@ def __init__(self, graph, **kwargs):
         Accepts the following additional arguments apart from the ones in
         `BasePlot`:
 
-        :param rotate_labels: Whether to rotate node labels.
-        :type node_color: `bool`
+        :param node_layout: which/whether (a) node layout is used,
+            either 'rotation', 'numbers' or None
+        :type node_layout: `string`
         """
-        self.rotate = kwargs.pop("rotate_labels", False)
+
+        # Store node layout
+        specified_layout = kwargs.pop("node_layout", None)
```

Comment:
On a related note, this is not technically a "node layout", but a "node label layout". This is the first place I'm noticing the use of `node_layout`, for clarity I'd like to rename this to `node_label_layout` please. I will mark out the other places where I'd like this changed.